tuf/client_rework/download.py

```diff
@@ -23,24 +23,13 @@
   metadata of that file.
 """
 
-
-# Help with Python 3 compatibility, where the print statement is a function, an
-# implicit relative import is invalid, and the '/' operator performs true
-# division.  Example:  print 'hello world' raises a 'SyntaxError' exception.
-from __future__ import (
-    absolute_import,
-    division,
-    print_function,
-    unicode_literals,
-)
-
 import logging
 import tempfile
 import timeit
+import urllib
```

Comment:
`from urllib import parse` maybe -- and then use parse.unquote() etc. Current code only works by accident since some other module happens to import the right thing